API/CopilotService/__init__.py

```diff
@@ -8,17 +8,19 @@
": [\"--name\", \"--resource-group\", \"--query\", \"--output\"], \"reason\": \"Get the storage account connection string. (connstr will be used in subsequent commands).\", \"example\": \"az storage account show-connection-string --name $storage --resource-group $resourceGroup --query connectionString --output tsv\"}, {\"command\": \"functionapp config appsettings set\", \"arguments\": [\"--name\", \"--resource-group\", \"--settings\"], \"reason\": \"Update function app settings to connect to the storage account.\", \"example\": \"az functionapp config appsettings set --name $functionApp --resource-group $resourceGroup --settings StorageConStr=$connstr\"}], \"Reason\": \"Create an Azure Function that connects to an Azure Storage\"}"}, {"role": "user", "content": "Please help me create a resource group with random resource name"}, {"role": "assistant", "content": "\"{\"Description\": \"Create a Resource Group\", \"CommandSet\": [{\"command\": \"resource group create\", \"arguments\": [\"--name\", \"--location\"], \"reason\": \"Create a resource group\", \"example\": \"az group create --name chatgpt-resourcegroup-1682324585 --location $location\"}], \"Reason\": \"Create a Resource Group by defining group name and location\"}"}, {"role": "user", "content": "I want to create a website with service and database"}, {"role": "assistant", "content": "{\"Description\": \"Connect an app to MongoDB (Cosmos DB).\", \"CommandSet\": [{\"command\": \"appservice plan create\", \"arguments\": [\"--name\", \"--resource-group\", \"--location\"], \"reason\": \"Create an App Service Plan\", \"example\": \"az appservice plan create --name $appServicePlan --resource-group $resourceGroup --location $location\"}, {\"command\": \"webapp create\", \"arguments\": [\"--name\", \"--plan\", \"--resource-group\"], \"reason\": \"Create a Web App\", \"example\": \"az webapp create --name $webapp --plan $appServicePlan --resource-group $resourceGroup\"}, {\"command\": \"cosmosdb create\", \"arguments\": [\"--name\", \"--resource-group\", \"--kind\"], \"reason\": \"Create a Cosmos DB with MongoDB API\", \"example\": \"az cosmosdb create --name $cosmosdb --resource-group $resourceGroup --kind MongoDB\"}, {\"command\": \"cosmosdb keys list\", \"arguments\": [\"--name\", \"--resource-group\", \"--type\", \"--query\", \"--output\"], \"reason\": \"Get the MongoDB URL (connectionString will be used in subsequent commands).\", \"example\": \"az cosmosdb keys list --name $cosmosdb --resource-group $resourceGroup --type connection-strings --query connectionStrings[0].connectionString --output tsv\"}, {\"command\": \"webapp config appsettings set\", \"arguments\": [\"--name\", \"--resource-group\", \"--settings\"], \"reason\": \"Assign the connection string to an App Setting in the Web App\", \"example\": \"az webapp config appsettings set --name $webapp --resource-group $resourceGroup --settings MONGODB_URL=$connectionString\"}], \"Reason\": \"Connect an app to MongoDB (Cosmos DB).\"}"}]"""
```

Comment:
Could we use a dedicated file to centrally store the configuration of default Prompt, such as the `prompt_config.py`

Reply:
What about moving the default Prompt in a separate PR?

Reply:
OK, sounds good!